repo/nirepo.yml

```diff
@@ -0,0 +1,1783 @@
+          schema:
+            type: object
+            description:  A required array of objects which describe which specific store items to replicate.
+            properties:
+              storeItemDescriptors:
+                type: array
+                items:
+                  $ref: '#/definitions/StoreItemDescriptor'
+      responses:
+        202:
+          $ref: '#/responses/JobIdResponse'
+        400:
+          $ref: '#/definitions/Error'
+        401:
+          $ref: '#/definitions/Unauthorized'
+        default:
+          $ref: '#/definitions/Error'
+  /v1/jobs:
+    get:
+      x-ni-privilege: Read
+      x-ni-operation: listJobs
+      tags: [jobs]
+      summary: List all jobs
+      description: Gets all "jobs" in the service. See the entry for "Get job details" route for more information about jobs.
+      parameters:
+        - in: query
+          name: id
+          type: string
+          required: false
+          description: A comma-delimited list of job ids.
+        - in: query
+          name: operation
+          type: string
+          required: false
+          description: A comma-delimited list of operations.
+        - in: query
+          name: status
+          type: string
+          required: false
+          description: A comma-delimited list of statuses.
+        - in: query
+          name: resourceId
+          type: string
+          required: false
+          description: A comma-delimited list of resource ids.
+      responses:
+        200:
+          description: The request has succeeded.
+          schema:
+            type: object
+            properties:
+              jobs:
+                type: array
+                items:
+                  $ref: '#/definitions/Job'
+        401:
+          $ref: '#/definitions/Unauthorized'            
+        default:
+          $ref: '#/definitions/Error'
+  /v1/jobs/{id}:
+    parameters:
+      - in: path
+        name: id
+        description: The id of this resource.
+        type: string
+        required: true
+        x-example: "5c5d49f2781e2b2cd045267b"
+    get:
+      x-ni-privilege: Read
+      x-ni-operation: getJobs
+      tags: [jobs]
+      summary: Get job details
+      description: Gets a job with a specific identifier.
+      responses:
+        200:
+          description: The request has succeeded. - If the job has not completed.
+          headers:
+            Location:
+              type: string
+              description: Contains the relative URI of the job that was created for tracking the status of the operation. Execute a GET against this URI to get information about the status of the operation.
+          schema:
+            type: object
+            properties:
+              job:
+                $ref: '#/definitions/Job'
+        303:
+          description: See Other if the job has completed, with the Location header set to the URI of the created or modified resource, if a resource was created or modified.
```

Comment:
If the job has completed, see Other with the Location header set to the URI of the created or modified resource.